frontend/src/components/eMIB/EditEmail.jsx

```diff
@@ -130,54 +145,100 @@ class EditEmail extends Component {
             <p className="font-weight-bold" style={styles.header.title}>
               {LOCALIZE.emibTest.inboxPage.addEmailResponse.selectResponseType}
             </p>
-            <div style={styles.header.radioButtonZone}>
-              <div style={styles.header.responseTypeRadio}>
+            <div style={styles.header.radioButtonZone} className="radio-button-hover">
```

Comment:
These should be grouped with a feildset: https://www.w3.org/WAI/tutorials/forms/grouping/

Reply:
added in fd4bebd